Vagrantfile

```diff
@@ -46,12 +47,13 @@ Vagrant.configure(2) do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  config.vm.synced_folder ".", "/vagrant",
+  config.vm.synced_folder ".", "/home/nico/",
```

Comment:
We shouldn’t be changing the /vagrant folder. By changing it to a users folder that would mean all users in the future would have to share it.